pkg/global/nucleusconstants.go

```diff
@@ -57,3 +59,20 @@ var NeuronHost string
 func SetNeuronHost(host string) {
 	NeuronHost = host
 }
+
+var LangArgKeyMap = map[string]string{
+	"pattern":          "--pattern",
+	"config":           "--config",
+	"diff":             "--diff",
+	"command":          "--command",
+	"locator-file":     "--locator-file",
+	"frameworkVersion": "--frameworkVersion",
+}
```

Comment:
I don't see a benefit of this map. Instead it is slightly increasing the brevity of the underlying code where this map is being used. If you have used meaningful constants this would have made sense.

Reply:
added them to constants.